Neo4j.Driver/Neo4j.Driver.sln

```diff
@@ -11,15 +12,17 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{622A94AD-BC4D-4162-AE43-A5CA055F24FC}"
 	ProjectSection(SolutionItems) = preProject
 		..\LICENSE = ..\LICENSE
+		..\Neo4j.Driver.Docs.shfbproj = ..\Neo4j.Driver.Docs.shfbproj
 		..\NOTICE = ..\NOTICE
 		..\README.md = ..\README.md
-		..\Neo4j.Driver.Docs.shfbproj = ..\Neo4j.Driver.Docs.shfbproj
 	EndProjectSection
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Neo4j.Driver.Tck.Tests", "Neo4j.Driver.Tck.Tests\Neo4j.Driver.Tck.Tests.csproj", "{E95B6563-4BC7-4B12-98C4-97FC8A8E5721}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Neo4j.Driver.Metrics", "Neo4j.Driver.Metrics\Neo4j.Driver.Metrics.csproj", "{F5F689DC-352C-4C0C-8F07-466CF7FBF7B8}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Neo4j.Driver.Logging", "Neo4j.Driver.Logging\Neo4j.Driver.Logging.csproj", "{A0808ECC-B2E4-4CC7-B4F0-6B353DB1156C}"
```

Comment:
Maybe name the project as `Neo4j.Driver.Extensions.Logging` to better express its scope?